tmt/steps/report/reportportal.py

```diff
@@ -36,50 +36,51 @@ class ReportReportPortalData(tmt.steps.report.ReportStepData):
         help="The launch name (name of plan per launch is used by default).")
     launch_description: Optional[str] = field(
         option="--launch-description",
-        metavar="LAUNCH_DESCRIPTION",
-        default=os.getenv('TMT_PLUGIN_REPORT_REPORTPORTAL_LAUNCH_DESCRIPTION'),
-        help="Pass the description for ReportPortal launch, especially with '--suite-per-plan' "
-             "option (Otherwise Summary from plan fmf data per each launch is used by default).")
+        metavar="DESCRIPTION",
+        default=os.getenv('TMT_PLUGIN_REPORT_REPORTPORTAL_DESCRIPTION'),
+        help="Pass the description for ReportPortal launch with '--suite-per-plan' option"
+             "or append the original (plan summary) with additional info."
+             "Appends suite/test description with upload-to-launch/suite options.")
     launch_per_plan: bool = field(
         option="--launch-per-plan",
-        default=False,
+        default=bool(os.getenv('TMT_PLUGIN_REPORT_REPORTPORTAL_LAUNCH_PER_PLAN')),
```

Comment:
`0` would be translated to `True`, which would be 1. puzzling, 2. incorrect. It's a non-empty string, therefore it's true-ish:

```
happz@multivac ~ $ export TMT_PLUGIN_REPORT_REPORTPORTAL_LAUNCH_PER_PLAN="0"
happz@multivac ~ $ python
Python 3.11.8 (main, Feb  7 2024, 00:00:00) [GCC 13.2.1 20231011 (Red Hat 13.2.1-4)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> bool(os.getenv('TMT_PLUGIN_REPORT_REPORTPORTAL_LAUNCH_PER_PLAN'))
True
>>> 
```

This should work:

```python
bool(os.getenv('TMT_PLUGIN_REPORT_REPORTPORTAL_LAUNCH_PER_PLAN') == '1')
```

Reply:
You're right. Thanks a lot.

Reply:
What do you think of this?
bool(os.getenv('TMT_PLUGIN_REPORT_REPORTPORTAL_LAUNCH_PER_PLAN') in ('True', 'true', '1'))